docs/modules/publisher-directmethods.md

```diff
@@ -0,0 +1,325 @@
+[Home](../../readme.md)
+
+### Configuration via IoT Hub Direct methods ###
```

Comment:
The document should start with a top level heading with one `#`. Please remove two `#`s from all headings if we want to keep the same hierarchy.